<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraTestControls</class>
 <widget class="QWidget" name="CameraTestControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>694</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <widget class="QPushButton" name="pushButton_connectVega">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Do image processing</string>
   </property>
   <property name="text">
    <string>连接NDI</string>
   </property>
  </widget>
  <widget class="QmitkToolTrackingStatusWidget" name="m_StatusWidgetVegaToolToShow" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>711</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>991</width>
     <height>541</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>探测位置重复性测试</string>
    </attribute>
    <widget class="QWidget" name="widget_5" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>811</width>
       <height>431</height>
      </rect>
     </property>
     <widget class="QTextBrowser" name="textBrowserLog">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>40</y>
        <width>256</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_printPoint">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>310</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>打印位置</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_clearPoint">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>350</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>清空位置</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_readPosition">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>读取位置（单位：mm）</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_calculateRepeatability">
      <property name="geometry">
       <rect>
        <x>281</x>
        <y>100</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>计算位置重复性</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_repeatability">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>130</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>误差(mm)：</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>探测距离偏差测试</string>
    </attribute>
    <widget class="QWidget" name="widget_10" native="true">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>20</y>
       <width>951</width>
       <height>431</height>
      </rect>
     </property>
     <widget class="QPushButton" name="pushButton_readEnd10fLine">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>读取线段端点 1（单位：mm）</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_readEnd2OfLine">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>201</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>读取线段端点 2（单位：mm）</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_calculateLength">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>10</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>计算线段长度（单位：mm）</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_calculateDeviationOfLength">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>90</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>计算距离偏差</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_DeviationOfLength">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>120</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>距离偏差</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_End1OfLine">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>211</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_End2OfLine">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>50</y>
        <width>201</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_length">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>50</y>
        <width>151</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_printEnd1">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>250</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>打印端点1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_printEnd2">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>280</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>打印端点2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_clearEnd1">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>340</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>清空端点1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ClearEnd2">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>370</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>清空端点2</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QmitkToolTrackingStatusWidget</class>
   <extends>QWidget</extends>
   <header location="global">QmitkToolTrackingStatusWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
